src/BlazorUI/Bit.BlazorUI/Components/Inputs/TextField/BitTextField.scss

```
@@ -136,28 +163,64 @@
+// Visually hidden, but still read out: the element only exists to give aria-describedby something to
+// point at.
+.bit-tfl-dsc {
+    border: 0;
+    padding: 0;
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    position: absolute;
+    white-space: nowrap;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)